src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,58 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+const Tabs = ({ links, listClassName, navClassName }) => {
+  return (
+    <nav className={classNames("p-tabs", navClassName)}>
```

Comment:
I think because this is the wrapping element we could just have `className` instead of `navClassName`.